base-theme/layouts/partials/header.html

```diff
@@ -23,17 +62,26 @@
     </div>
     <div class="right">
       {{ block "extraheader" . }}{{ end }}
-      <a class="d-flex give-button-header py-2 px-3" href="https://giving.mit.edu/give/to/ocw/?utm_source=ocw&utm_medium=homepage_banner&utm_campaign=nextgen_home">
+      <a
+        class="d-flex give-button-header py-2 px-3"
+        href="https://giving.mit.edu/give/to/ocw/?utm_source=ocw&utm_medium=homepage_banner&utm_campaign=nextgen_home"
+      >
         <span class="font-weight-bold w-100">Give now</span>
       </a>
       <a href="/about">
-        <div class="text-white font-weight-bold w-100 px-3 pl-5 py-2">About OCW</div>
+        <div class="text-white font-weight-bold w-100 px-3 pl-5 py-2">
+          About OCW
+        </div>
       </a>
       <a class="coming-soon">
-        <div class="text-white font-weight-bold w-100 px-3 py-2 coming-soon">help & faqs</div>
+        <div class="text-white font-weight-bold w-100 px-3 py-2 coming-soon">
+          help & faqs
```

Comment:
I know you didn't change this, but the capitalization here (and w/ `'contact us'` below) seems inconsistent, maybe we should go for

- `"Help & FAQs"`
- `"Contact us"`

@pdpinch any input on that?

Reply:
Thanks for pointing it out Alice.

All the desktop menu items have been made uppercase via the following css: 
![image](https://user-images.githubusercontent.com/93309234/148177142-d9749be5-2e31-4ce3-a0e4-19f701ebda87.png)

I tried to make all the mobile menu items upper case as well but they were not much appealing (as shown below) and I also didn't have the permission to view the sandwich menu design so not sure what is there in the design.
![image](https://user-images.githubusercontent.com/93309234/148177819-3e302d17-706b-4f95-9e22-e37626aec00b.png)

I can change it to any way which is appropriate.
Let me know @pdpinch 

Thanks

Reply:
haha, right you are :) when you are looking just at the source you can miss things!